migration-guides/0.15-Migration-Guide.md

```diff
@@ -257,3 +265,48 @@ Basic usage:
 parcel build index.html --no-source-maps --no-optimize --no-scope-hoist --dist-dir "dist/" # bundle for the browser
 parcel build index.html --no-source-maps --dist-dir "dist/" # minified bundle for the browser
 ```
+
+## Breaking Changes Made in Core Libraries
+
+### Changes affecting multiple libraries
+
+- Migrated all FFI to ES modules and dropped support for CommonJS modules.
+- Removed all kind-specific Proxy types (e.g. `SProxy`, `Proxy2`, `Proxy3`, `RLProxy`, etc.)
```

Comment:
Maybe note "If you are using any of these types, replace them with `Proxy` instead."